.github/workflows/round-robin.yml

```diff
@@ -0,0 +1,27 @@
+name: Reviewer Round Robin
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  RoundRobin:
+    env:
+      IN_MAINTAINER_LIST: ${{ contains( secrets.MAINTAINER_LIST, github.actor) }}
+    if: ${{ ( github.event.pull_request.assignee == null ) && ( env.IN_MAINTAINER_LIST == 'true' ) }}
+    runs-on: ubuntu-latest
+    name: 'Round Robin'
+    steps:
+      - name: 'Round Robin Step'
+        uses: justinretzolk/round-robin@v1
+        with:
+          allow_bootstrap: true
+          maintainers: '["anGie44", "ewbankkit", "gdavison", "johnsonaj", "YakDriver", "zhelding"]'
```

Comment:
Does this action guard against assigning someone's PR to themself?

Reply:
It does not! I'd not considered that 🤦. I'll work on getting that fixed before we merge this.